### PR TITLE
add homework from lesson 30

### PR DESCRIPTION
**Pull request by yurii-polych**
Homework 30

### CODE REVIEW

File: 30/hw-30.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+sites = [
+        'https://www.google.com/',
+        'https://www.facebook.com/',
+        'https://twitter.com/',
+        'https://www.amazon.com/',
+        'https://www.apple.com/'
+        ]
+
+def get_random_site(lst):
+        return random.choice(lst)
```
Comment (acman) [inline 30/hw-30.py]: nit: too much spaces here

File: 30/hw-30.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+res = requests.get(get_random_site(sites))
+print(f'Status code: {res.status_code}, url: {res.url}, HTML length: {len(res.text)}')
+
+
+# Task 2. ====================================
+try:
+    city = input('Enter your city: ')
+    coords_url = 'https://geocoding-api.open-meteo.com/v1/search'
+    city_coords = requests.get(f'{coords_url}?name={city}&count=1')
+    data = city_coords.json()
```
Comment (acman) [inline 30/hw-30.py]: nit: better to create functions for all logic parts like:
`get_city_coordinates`

File: 30/hw-30.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    latitude = round(data['results'][0].get('latitude'), 2)
+    longitude = round(data['results'][0].get('longitude'), 2)
+except KeyError:
+    print('The city you entered is not available. Enter correct name next time.')
+else:
+    meteo_url = 'https://api.open-meteo.com/v1/forecast'
+    current_weather = requests.get(f'{meteo_url}?latitude={latitude}&longitude={longitude}'
```
Comment (acman) [inline 30/hw-30.py]: nit: better to use requests `params` for setting multiple query parameters

**Review by acman — APPROVED** [review]
LGTM!

File: 30/hw-30.py (hunk 2 above)
Comment (yurii-polych) [reply 30/hw-30.py]: Fixed

File: 30/hw-30.py (hunk 3 above)
Comment (yurii-polych) [reply 30/hw-30.py]: Fixed

**Pull request merged**